slowquant/qiskit_interface/linear_response/allprojected.py

```diff
@@ -36,14 +36,15 @@ def run(
         print("Gs", self.num_G)
         print("qs", self.num_q)
 
-        # This hamiltonian is expensive and not needed in the naive orbitale rotation formalism
-        self.H_2i_2a = hamiltonian_pauli_2i_2a(
-            self.wf.h_mo,
-            self.wf.g_mo,
-            self.wf.num_inactive_orbs,
-            self.wf.num_active_orbs,
-            self.wf.num_virtual_orbs,
-        )
+        if self.num_q != 0:
+            # This hamiltonian is expensive and not needed in the naive orbitale rotation formalism
```

Comment:
Here we inside the file allprojected.py?

I.e. this is never naive orbital rotations?

Reply:
You could - in principle - do allproj LR within complete space. Yes it is just projLR. This is just for completeness here. 

The problem were before that in standard setting, do_RDM = True,  if you were working in complete space (no q's) you were still calculating the RDMs. This is now avoided